tree/ntuple/v7/src/RMiniFile.cxx

```diff
@@ -950,7 +952,10 @@ ROOT::Experimental::RNTuple ROOT::Experimental::Internal::RMiniFileReader::GetNT
       }
       offset = offsetNextKey;
    }
-   R__ASSERT(found);
+   if (!found) {
+      return R__FAIL("no NTuple named '" + std::string(ntupleName)
```

Comment:
`NTuple` --> `RNTuple` (the name ntuple without the `R` has already a meaning within ROOT.  It's a certain kind of a tree.)

Reply:
Thank you, done in e11ae583cef366afa8282767965f9c9a84d51dea